aurelia-bootstrap-select/src/abp-select.js

```diff
@@ -455,12 +455,6 @@ export class AbpSelectCustomElement {
     this.domElm.on('changed.bs.select', (e, clickedIndex, newValue, oldValue) => {
       this.selectedValue = this.domElm.selectpicker('val');
       let selection = this.findItems(this.collection, this.selectedValue, this.objectKey);
-      if (selection.indexes) {
-        this.domElm.selectpicker('val', selection.indexes);
-      }
-
-      // refresh the bindable value/item
-      this.selectedValue = selection.indexes;
```

Comment:
what is the reason in removing this piece? At first glance, this looks like I won't have the `selectedValue` refereshed, which is a regression

Reply:
Without removing this code, `newValue` and `oldValue` would be the same in `selectedValueChanged` and the `selectedValue` would end up being an array in case of single selection. I did not understand exactly why this is the case.

With respect to the deleted code, the selected value is already updated in line 456. And by setting `this.selectedItems`, `this.selectedValue` will be updated again in `selectedItemChanged` as far as I can see.

I have only tested this with single selection so you might want to test it with multiple selection.

Reply:
technically, you can test it quickly through the GitHub [demo page](https://ghiscoding.github.io/Aurelia-Bootstrap-Plugins/#/bootstrap-plugins/bootstrap-select). On top of each dropdown, I displayed the "Value" and "Item". If your change still shows both of these for all the demo dropdowns, then I'm fine with your code.

Reply:
Can you provide some instructions on how to run this locally with the modified version?

Reply:
Sure the GitHub demo is actually part of this repo, you just have to go under the [doc](https://github.com/ghiscoding/Aurelia-Bootstrap-Plugins/tree/master/doc) folder and do an `npm install`, these are the steps I do before releasing, it's not the most effective way but it works
1. npm install GitHub `doc` demo
2. run `gulp build` on the lib we are updating (in our case `aurelia-bootstrap-select`)
3. copy the `dist` folder content into the `doc/node_modules/aurelia-bootstrap-select/`
   - you need to delete the content before pasting the `dist` content in that folder

Reply:
Ok got it. The functionality looks good! Please have a look yourself - just to be sure. ;)

Reply:
Thanks I'll do that in the coming days 👍